src/cli.js

```diff
@@ -26,27 +27,27 @@ if (!isAscii(homeDir) || containsSpace(homeDir)) {
 analytics.setArgv(process.argv);
 
 (async () => {
-  if (await autoUpdate()) {
+  if (await autoUpdate(cliArgs)) {
     return null;
   }
-  await authorizeRequests(await getRefreshToken());
+  const refreshToken = await getRefreshToken();
+  authorizeRequests(refreshToken);
```

Comment:
This call was awaited before. Please check if this call should still be awaited and add `await` if required.